### DISCUSSION TITLE
[Feature Request] Keyboard Shortcuts

### DISCUSSION THREAD

**Discussion by vpoguru** [GrandOrgue/LoopAuditioneer#6] [Ideas]
Keyboard shortcuts can significantly enhance efficiency when working with LoopAuditioneer. Currently, “Open previous file” is mapped to Option-Command-U, and “Open next file” to Option-Command-N on a Mac.

Typically, shortcuts like (U)p and (D)own, or (P)revious and (N)ext, are used for navigation, the combination of U and N is quite unusual.

Maybe it is even possible to use P and N even without modifier key.

**Comment by larspalo** [comment]
> the combination of U and N is quite unusual.

Well... In Swedish U would stand for "Upp" (Up) and N for "Ner" (Down). I've got no better explanation, nor can I say that it's perfect. I will take a change to Ctrl+Alt+P for (P)revious in consideration as it might indeed match the language better.

> Maybe it is even possible to use P and N even without modifier key.

It's certainly possible, but I've refrained from using single keys for such operations as they will immediately discard any (potentially unsaved) work and close the file. See it as a kind of "safety" mechanism as pressing/holding down multiple keys requires a bit more thought to perform. The single keys are only used for such things that won't cause a sudden "surprise" for the user in regard to what the state is for the file on disc is vs the file in memory.
